tests/views/test_registry_instrument_view.py

```diff
@@ -0,0 +1,206 @@
+import base64
+import json
+from pathlib import Path
+from unittest.mock import patch
+
+from flask import current_app
+
+from tests.test_client import TestClient
+
+api_root = "/collection-instrument-api/1.0.2"
+exercise_id = "3ff59b73-7f15-406f-9e4d-7f00b41e85ce"
+form_type_exists = "0001"
+form_type_does_not_exist = "9999"
+
+
+class TestRegistryInstrumentView(TestClient):
+    """
+    Registry Instrument view unit tests
+    """
+
+    def test_get_registry_instrument_returns_200_and_object(self):
+        with patch(
+            "application.views.registry_instrument_view."
+            "RegistryInstrument.get_registry_instrument_by_exercise_id_and_formtype"
+        ) as mock_get_registry_instrument_by_exercise_id_and_formtype:
+            with open(Path(__file__).parent.parent / "test_data" / "registry_instrument.json") as f:
+                mock_get_registry_instrument_by_exercise_id_and_formtype.return_value = json.load(f)
+            response = self.client.get(
+                f"{api_root}/registry-instrument/exercise-id/{exercise_id}/formtype/{form_type_exists}",
+                headers=self.get_auth_headers(),
+            )
+            self.assertStatus(response, 200)
+            self.assertEqual(len(response.json), 8)  # The 8 fields of a registry instrument
```

Comment:
I think we need to assert the content not the size here

Reply:
Can do.. this time I _was_ being lazy !

Reply:
Added an assertion of the source object to cover the content of each attribute. 

I admit it now appears overkill, but but going to leave the size assertion in place for visibility of the (very strict) object we're handling. It will also highlight the tests need tweaking if we independently extend the object in the future.